recipes/install-kibana.rb

```diff
@@ -11,49 +11,24 @@
 
 elk_info = get_elk_info
 
-kibana_version = elk_info[:kibana_version]
-kibana_checksum = elk_info[:kibana_checksum]
-download_path = "#{::Chef::Config[:file_cache_path]}/kibana.tar.gz" 
+kibana_major_version = elk_info['kibana_major_version']
+kibana_repo_uri = elk_info['kibana_repo_uri']
 
-create_ssl_cert(elk_info[:http_ssl])
+create_ssl_cert(elk_info['http_ssl'])
 
-group 'kibana' do
-  append true
+apt_repository 'kibana' do
+  uri kibana_repo_uri
+  components ['stable', 'main']
+  keyserver 'ha.pool.sks-keyservers.net'
+  key '46095ACC8548582C1A2699A9D27D666CD88E42B4'
```

Comment:
right. they're public keys for the package repos. needed when the underlying chef code runs apt-add-repository.